.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "lib/finediff"]
+	path = lib/finediff
+	url = https://github.com/gorhill/PHP-FineDiff
```

Comment:
As shyim already mentioned, this library hasn't been updated in ages. At this point I'd prefer to just completely migrate it into this repository instead. I think that would make this plugin future proof and it would also enable you to add a proper namespace and get rid of the require_once in the constructor of class Diff.

Reply:
Sounds good. Any hints regarding the license under which finediff is released?

Reply:
@kleinmann I already saw that but I am not sure what changes are fine. Ain't no attorney.